gensim/models/deprecated/word2vec.py

```diff
@@ -162,9 +162,6 @@
 
 logger = logging.getLogger(__name__)
 
-
-# failed... fall back to plain numpy (20-80x slower training than the above)
-FAST_VERSION = -1
```

Comment:
We may want to keep `FAST_VERSION`: it's in all tutorials and instructions, all around the web.

Also, its value is useful in debugging. Even with optimizations (`FAST_VERSION >= 0`), it's sometimes good to know if FAST_VERSION is 0, 1, or 2.

Although I'm not sure what this `deprecated/word2vec.py` is.